Gemfile

```diff
@@ -62,6 +62,7 @@ gem 'kaminari', '~> 1.2'
 gem 'link_header', '~> 0.0'
 gem 'mario-redis-lock', '~> 1.2', require: 'redis_lock'
 gem 'mime-types', '~> 3.6.0', require: 'mime/types/columnar'
+gem 'mutex_m'
```

Comment:
The `httpclient` gem needs this but does not depend on it.

Separately, that gem looks very unmaintained. I see there's a shopify fork. Maybe something to check out.

Reply:
I am not sure where we use this gem, maybe this should indeed be replaced entirely?

Reply:
It's an indirect dependency, pulled in by a few gems.